packcheck.sh

```diff
@@ -381,7 +381,7 @@ show_help() {
   help_envvar STACK_BUILD_OPTIONS "ADDITIONAL stack build command options to append"
 
   show_step1 "cabal options"
-  help_envvar CABAL_PROJECT "Alternative cabal project config, cannot be a path, just the file name"
+  help_envvar CABAL_PROJECT "Alternative cabal project file, path relative to project root"
```

Comment:
Same question asked w.r.t `stack.yaml`.
Looks like I'm missing something. Need to go through the code once.